scripts/tasks/check-and-rewrite-package-json.js

```diff
@@ -81,6 +81,8 @@ for (const dir of directories) {
         scripts: {
             build: 'rollup --config ../../../scripts/rollup/rollup.config.js',
             dev: 'rollup  --config ../../../scripts/rollup/rollup.config.js --watch --no-watch.clearScreen',
+            test: name === '@lwc/ssr-compiler' ? undefined : 'jest ./src/**/*.spec.ts',
+            'test-manual': name === '@lwc/ssr-compiler' ? 'jest ./src/**/*.spec.ts' : undefined,
```

Comment:
I think I'll just remove this. It was a convenience for me when I was working in the `@lwc/ssr-compiler` subdirectory, but it isn't important and I agree it is gross.

Reply:
Do we even need individual `"test"` scripts for each `package.json`? You can test in the root directory like so:

```bash
npx jest --config ./scripts/jest/root.config.js ./packages/@lwc/whatever
```

Reply:
Yes, that's what Will was pointing out too. There's no need for it, it was a convenience for me as I was putting the PR together, and it can be removed now.